ReadMe.md

```diff
@@ -0,0 +1,20 @@
+# React DOM Lite
+
+_"Perfection is achieved, not when there is nothing more to add, but when there is nothing left to take away.” - Antoine de Saint-Exupery._
+
+Compliance and amazing cross browser support led to a robust but heavy `react-dom`. React DOM Lite is an attempt to sculpt away some of the mass away and see if we can make something more low powered device friendly.
+
+By no means, this is a pursuit for the perfect library. It's about finding out what it is that we can live without in `react-dom`?
+
+## Road map
+
+Keeping in mind the existing React ecosystem (and of course the web ecosystem too), following is the feature list to attain feature parity with the existing reac-dom:
+
+* SVG and namespaced attribute support
+* Event normalization / polyfilling
+* Portals (event propagation)
+* Controlled inputs
+* Browser support matrix
+* SSR, hydration.
+
+Notwithstanding the list above, feature partiy is something to be taken with a grain of salt: browser support, event polyfilling, controlled components etc. are being currently being evaluated to be toned down, removed or retained.
```

Comment:
maybe "The goal is to be compatible with the react ecosystem, while remaining lite. This will likely mean that the supported browsers, will be more limited than react-dom, and attempts to polyfill differences between browsers will be limited and more tightly scoped.

Reply:
Add it or replace it? Do you feel the it currently doesn't convey the same meaning?